### ISSUE TITLE
Double @ when using compliment for usernames in slack

### ISSUE THREAD

**Issue by toddnguyen** [benderTheCrime/hubot-emergency-compliment#1]
Right now, when you use the compliment command on a slack username with an @ character as the first character hubot will add another @ to the name.

Example:

```
me>> hubot compliment @user1
hubot>> @@user1, you look good today
```

**Comment by benderTheCrime** [comment]
Hey, @toddnguyen 

Thanks for letting me know about this! I legitimately had no idea anyone was using this plugin. I made a PR for this, wrote some really terrible tests to validate that it is now working properly. Let me know what you think: https://github.com/benderTheCrime/hubot-emergency-compliment/pull/2

**Comment by toddnguyen** [comment]
I checked it out, looks good to me!

**Comment by benderTheCrime** [comment]
This is fixed in version 1.1.2: https://github.com/benderTheCrime/hubot-emergency-compliment/releases/tag/1.1.2. Updating should fix that for you!

Thanks for reporting this
